src/components/Navbar/index.tsx

```diff
@@ -220,16 +216,16 @@ export class NavbarBase extends React.Component<Props, State> {
           </div>
         </Navbar.Brand>
         <Navbar.Text className={styles.text}>
-          {currentVersion && file ? (
+          {currentVersionId ? (
             <a
               className={styles.legacyLink}
               href={
-                baseFileId
-                  ? `${baseUrlToLegacy}/compare/${file.id}...${baseFileId}/${
-                      path ? `file/${path}` : ''
+                currentBaseVersionId
+                  ? `${baseUrlToLegacy}/compare-redirect/${currentVersionId}...${currentBaseVersionId}/${
```

Comment:
So why would we use a (default) prop for `reviewersHost` and not for the api lang? Both are env vars and behave the same.

Reply:
There doesn't seem to be a good reason for this. I guess it evolved over time, as `reviewersHost` was added first, before there was a need for api lang, and then when the latter was added it got added in a different manner. I can change it so they are both props.